apps/calcom/config.json

```diff
@@ -0,0 +1,106 @@
+{
+    "name": "Cal.com",
+    "available": true,
+    "port": 8294,
+    "exposable": true,
+    "id": "calcom",
+    "description": "The open source Calendly successor. You are in charge of your own data, workflow, and appearance.\nCalendly and other scheduling tools are awesome. It made our lives massively easier. We're using it for business meetings, seminars, yoga classes, and even calls with our families. However, most tools are very limited in terms of control and customization.\n    That's where Cal.com comes in. Self-hosted or hosted by us. White-label by design. API-driven and ready to be deployed on your own domain. Full control of your events and data.",
+    "tipi_version": 1,
+    "version": "3.7.9",
+    "categories": ["calendar", "social"],
+    "short_desc": "Scheduling infrastructure for absolutely everyone.",
+    "author": "cal.com",
+    "source": "https://github.com/calcom/cal.com",
+    "website": "https://cal.com/",
+    "supported_architectures": ["amd64"],
+    "form_fields": [
+        {
+            "type": "password",
+            "label": "Next.js Auth secret. Generate one with `openssl rand -base64 32`",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "CALCOM_NEXTAUTH_SECRET"
+        },
+        {
+            "type": "password",
```

Comment:
We can use a `random` type here and give it a `min` value of 50 to generate it automatically